server/plugin/http.go

```diff
@@ -66,19 +65,7 @@ func (c *client) Call(method, path, contentType string, inBody io.Reader, out in
 
 	resp, err := c.httpClient.Do(req)
 	if err != nil {
-		if strings.Contains(err.Error(), "invalid character '<'") {
-			return nil, http.StatusInternalServerError, ErrorContentTypeNotJSON
-		}
-
-		updatedError := strings.ReplaceAll(err.Error(), c.plugin.getConfiguration().ServiceNowBaseURL, "")
-		if strings.Contains(err.Error(), "dial tcp") {
-			errorData := strings.Split(c.plugin.getConfiguration().ServiceNowBaseURL, ":")
-			if len(errorData) == 3 {
-				updatedError = strings.ReplaceAll(updatedError, errorData[2], "")
-			}
-		}
-
-		return nil, http.StatusInternalServerError, errors.New(updatedError)
+		return nil, http.StatusInternalServerError, ErrorConnectionRefused
```

Comment:
in the comment they also said to log the error as well.Where you have logged the error ?

Reply:
it is being done in the api.go ex: https://github.com/Brightscout/mattermost-plugin-servicenow/blob/3e286e9219279ff77de74ef9cc95d58ca8d878da/server/plugin/api.go#L461

Reply:
callJson and call function is getting called from multiple places do we have a need to log those errors everywhere or this alone is enough